src/Tizen.NUI/src/internal/EXaml/Action/AddToResourceDictionary.cs

```diff
@@ -64,7 +64,7 @@ public void Init()
 
         public void OnActive()
         {
-            if (null != childOp)
+            if (null != childOp && null != (childOp.ValueList[0] as Instance))
```

Comment:
as following myroot's review, shouldn't this be changed like the below?

```suggestion
            if (null != childOp && childOp.ValueList.Count > 3 && (childOp.ValueList[0] is Instance instance))
            {
                int instanceIndex = instance.Index;
```

Reply:
> as following myroot's review, shouldn't this be changed like the below?

DONE